test/unit/mdc-textfield/mdc-text-field.test.js

```diff
@@ -86,16 +86,23 @@ class FakeOutline {
   }
 }
 
-test('#constructor when given a `mdc-text-field--box` element instantiates a ripple on the root element', () => {
+test('#constructor when contains only `mdc-text-field` element instantiates a ripple on the root element', () => {
```

Comment:
nit: rename this test `#constructor instantiates a ripple on the root element by default`

Reply:
Got it!